great_expectations/render/renderer/content_block/bullet_list_content_block.py

```diff
@@ -143,12 +106,22 @@ def expect_column_unique_value_count_to_be_between(cls, expectation, styling=Non
 
         if (params["min_value"] is None) and (params["max_value"] is None):
             template_str = "may have any number of unique values."
-        elif params["min_value"] is None:
-            template_str = "must have fewer than $max_value unique values."
-        elif params["max_value"] is None:
-            template_str = "must have more than $min_value unique values."
         else:
-            template_str = "must have between $min_value and $max_value unique values."
+            if params.get("mostly"):
```

Comment:
This is a tiny issue, but I'm confused why we're using `if params.get("mostly"):` here and `if params["min_value"] is None:` everywhere else.

Is the behavior the same?

Since we're using `substitute_none_for_missing`, is it even possible for `mostly` to be missing?

Reply:
@roblim Reviewing this more closely, it looks like you've replaced my `if 'mostly' in params` with `if params.get("mostly"):`

This is an improvement, but the problem is: my original logic itself was incorrect.

Because we're calling `substitute_none_for_missing` at the top of each method, we *know* that `mostly` will be present.

Instead of checking whether it exists in the dict, we should always be checking `if mostly == None`. (Also, this means that we ALWAYS have to be disciplined about using `substitute_none_for_missing` and including ALL possible parameters for each method.)

Reply:
K. Changed to "if params["mostly"] is not None". Regarding your first comment about refactoring `PrescriptiveBulletListContentBlockRenderer`, that refactoring has been done but is in a different branch because it's for prescriptive rendering of EVRS.